### ISSUE TITLE
Video accompanying this repo

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/learn-elixir#63]
After the discussion in #56 about there not being a free/bought video series for Elixir (there is a subscription based one on codeschool.io, and a free intro, but both are very basic, and the bought one on udemy, but see the discussion in #56 ) I think it would be great (after a suggestion from @iteles) to have a video/series of videos accompanying this repo! 

I'm not sure how we could actualise this, but I'd love to be involved, any ideas?

**Comment by markwilliamfirth** [comment]
Completely achievable!

Initial steps:

- [ ] Decide the purpose and audience for this video

- [ ] Compile information / learning content suited to this purpose and audience

- [ ] Decide how you want to show this information? Partial talking and partial screen capture possibly? Possibly slides?

- [ ] Decide how to put this into video format - 1 video or a playlist / series? (depends on the amount of content you want to share)

- [ ] Make a draft of the video script

- [ ] Review this script with @Cleop once complete (Cleo can give you valuable advice)

- [ ] After script has been agreed record yourself with camera and mic reading the script (ask Cleo / Sohil for help if you need it)

- [ ] Use a screen capture app to capture any coding etc you want to show + any additional content e.g. slides

- [ ] Put these together to form a draft of the video

- [ ] Review this with @Cleop and @markwilliamfirth

- [ ] Iterate based on feedback

- [ ] Post to repo and dwyl YouTube account!

**Comment by YvesMuyaBenda** [comment]
@markwilliamfirth  The man with the plan! Apologies, I could not resist!

**Comment by iteles** [comment]
@Cleop @sohilpandya Mentioning you guys as you've already produced the excellent intro vids and will certainly have some insights!
https://drive.google.com/open?id=0ByzPFwE23Wu2Y1lfQW1WVGV1Z3c
https://drive.google.com/open?id=0ByzPFwE23Wu2ZnBHUHNWSndLRHc

**Comment by ZooeyMiller** [comment]
Awesome info! I've just seen https://github.com/dwyl/learn-phoenix-framework/issues/100 in the `learn-phoenix-framework` repo, which links to a [elixir tutorial series on youtube by tensor programming](https://www.youtube.com/watch?v=R8CeZazrDHo). I'm going to check out these videos, but if these are comprehensive, do we as dwyl still want to create our own videos? Or should we just link to the already existing ones?

**Comment by ZooeyMiller** [comment]
Currently giving the videos I linked in https://github.com/dwyl/learn-elixir/issues/63#issuecomment-325591145 a watch to see if I want to make a PR linking to them in the README.

**Comment by ZooeyMiller** [comment]
So these videos (I've watched the first 5, after that he moves on to phoenix) could make a good addition to this repo, but I don't think they are good as a straight introduction to Elixir.

The videos moves very quickly from the bare basics into things about spawning processes and passing information around (which was new to me!) and I feel like this info is more suited to someone who has had a bit of an introduction to Elixir already. The videos don't even show the pipe operator, so I think perhaps they fill more the role of "once you've got to grips with the syntax and played around a bit, check these out", rather than a straight intro, so the need for dwyl's own videos is still there.

**Comment by finnhodgkin** [comment]
Yeah I agree with @ZooeyMiller. While some of the content is great, it's a strange mix of beginner and advanced topics. 

The videos are also very weirdly structured—sometimes they cut to a new screen without warning, and he covers things like spawning processes _before_ Elixir installation.

**Comment by ZooeyMiller** [comment]
I've made a repo in my personal github to plan these videos re: some of the stuff @markwilliamfirth mentioned, please take a look at the issues I've raised and add your input! https://github.com/ZooeyMiller/elixir-video-plan

**Comment by markwilliamfirth** [comment]
@ZooeyMiller there is also a video repo (https://github.com/dwyl/video) should you ever wish to use it

**Comment by iteles** [comment]
@ZooeyMiller @finnhodgkin Great feedback on these. 

I would encourage you to add the video issue to this repo rather than a separate one as they are all pertinent to people who are interested in the content we have here and there are 2 or 3 active people on this repo who could have some good feedback on these, other than just dwylers 😊

**Comment by ZooeyMiller** [comment]
@iteles Good point, apologies, I'm always hesitant to raise loads of issues in a repo with a lot of watchers but I will add those issue here instead to give everyone a chance to contribute.

**Comment by Cleop** [comment]
@ZooeyMiller looking forward to talking later.

FYI - check out the process/conversation that led to the creation of the tachyons bootstrap videos here: https://github.com/dwyl/tachyons-bootstrap/issues/9, in particular you may find the script structures useful. As @markwilliamfirth said, there was also conversation for these videos on the video repo too.
